planner/core/plan_cost_ver2.go

```diff
@@ -1021,8 +1027,10 @@ func cols2Exprs(cols []*expression.Column) []expression.Expression {
 }
 
 type costTrace struct {
+	name        string
 	factorCosts map[string]float64 // map[factorName]cost, used to calibrate the cost model
 	formula     string             // It used to trace the cost calculation.
+	costParams  map[string]interface{}
```

Comment:
The two new fields are not used.
Are they also used in your version?

Reply:
it's from gaosong' s code，I thought to change cost_ver2 to add formula. Will remove it.